drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -299,6 +400,23 @@ local function assign_child_profile(device, child_ep)
   return profile or "switch-binary"
 end
 
+local function do_configure(driver, device)
+  local energy_eps = device:get_endpoints(clusters.ElectricalEnergyMeasurement.ID)
```

Comment:
I think we need to use `embedded_cluster_utils.get_endpoints` in case the embedded cluster definition is used?

Reply:
thanks, good catch :+1: